tools/mkhtml.py

```diff
@@ -311,6 +313,27 @@ def get_addon_path(pgm):
 
 if not re.search('<html>', src_data, re.IGNORECASE):
     tmp_data = read_file(tmp_file)
+    """
+    Adjusting keywords html pages paths if add-on html man page
+    strored on the server
```

Comment:
```suggestion
    stored on the server
```

Reply:
Fixed it in [1549525](https://github.com/OSGeo/grass/pull/1165/commits/154952571dd087445bccb7e023c37384d83fcc93).